Source/Core/DolphinQt/GCMemcardCreateNew.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+
+class QComboBox;
+class QRadioButton;
+
+class GCMemcardCreateNew : public QDialog
```

Comment:
I'd add "Dialog" to the name to make it obvious what this is.

```suggestion
class GCMemcardCreateNewDialog : public QDialog
```